src/frontend/src/pages/ProjectPage/index.tsx

```diff
@@ -4,13 +4,15 @@ import ContractsOverviewPage from './pages/ContractsOverviewPage';
 import AllocateContractPage from './pages/AllocateContractPage';
 import ContractPage from './pages/ContractPage';
 import ScopedSwitch from '../../components/ScopedSwitch';
+import ManagePersonellPage from './pages/ContractPage/pages/ManagePersonnelPage';
 
 const ProjectPage = () => {
     return (
         <ScopedSwitch>
             <Route path="/" exact component={ContractsOverviewPage} />
             <Route path="/allocate" exact component={AllocateContractPage} />
             <Route path="/:contractId" component={ContractPage} />
+            <Route path="/:contractId/managedpersonnel" component={ManagePersonellPage} />
```

Comment:
This should probably be in the ContractPage, not project page to get the navigation? Also: /managedpersonnel or /managepersonnel?